dd-sdk-android/apiSurface

```diff
@@ -36,6 +36,10 @@ object com.datadog.android.DatadogEndpoint
   const val TRACES_EU: String
   const val RUM_US: String
   const val RUM_EU: String
+  const val NTP_0: String
+  const val NTP_1: String
+  const val NTP_2: String
+  const val NTP_3: String
```

Comment:
Do we need to expose those in public interface? 🤔

Reply:
Well they are part of the [official documentation](https://docs.datadoghq.com/integrations/ntp/), so it doesn't hurt to expose them.